public_html/app/models/Post.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+    class Post {
+
+        private $db;
+        public function __construct() {
+            $this->db = new Database;
+        }
+
+        public function getPosts() {
+            $this->db->query('
+                SELECT *,
+                posts.id as postId,
+                users.id as userId,
+                posts.created_at as postCreated,
+                users.created_at as userCreated
+                FROM posts
+                INNER JOIN users
+                ON posts.user_id = users.id
+                ORDER BY posts.created_at DESC
+                ');
+
+            $results = $this->db->resultSet();
+
+            return $results;
+        }
+
+        public function addPost($data){
+            // First we prepare the statement with query() function
+            $this->db->query('INSERT INTO posts (title, user_id, body) VALUES (:title, :user_id, :body)');
+            // Then we bind the values to the parameters
+            $this->db->bind(':title', $data['title']);
+            $this->db->bind(':user_id', $data['user_id']);
+            $this->db->bind(':body', $data['body']);
+            // Then we insert the into DB and call the execute method
+            if ($this->db->execute()) {
+                return true;
+            }
+            else {
+                return false;
+            }
+        }
+
+        public function getPostById($id) {
```

Comment:
getById(); Your model name is Post so you don't need put the same prefix to functions